packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -31,6 +31,7 @@ export interface IFabricTestLedgerV1ConstructorOptions {
   publishAllPorts: boolean;
   imageVersion?: string;
   imageName?: string;
+  fabricVersion?: string;
```

Comment:
Idea to extend this and make it more generic for the future uses: 
1. Rename the `fabricVersion` parameter to envVars
2. Change the parameter type to a `Map<string, string>` where the key is the variable name and the value is the ENV variable's value
3. The default value (`DEFAULT_OPTS`) can be a new map populated with the same information as before but this way: 

```typescript
envVars: new Map([ 
    [
        "fabricVersion", 
         "1.4.8"
    ] 
]),
```
4. In the start method just iterate over all map entries and convert them into an array of strings that are built in the expected format e.g. `${key}=${value}`, then just pass this in directly to dockerode as the `Env` property of the container.

Reply:
Incorporated the changes. Please have a re-look.